### ISSUE TITLE
Questions regarding your paper

### ISSUE THREAD

**Issue by gpastal24** [XunshanMan/MVGFormer#8]
Hello,
Your paper was a great read, interesting stuff.

However I have a couple of questions.

1.  During the iterative update of the 2d and 3d joints, is the root joint stationary or does it move? The reason I wonder this is the significant performance degradation when you use 128 T poses at the initial step. 
2. Your framework runs at ~5FPS. If you use 128 queries, how much faster does it get?
3. Have you tried training with voxel rotations yourself, as a data augmentation technique? You don't claim you do, you actually cite the paper " Learnable Triangulation of Human Pose" but I am curious.

Thank you!

**Comment by XunshanMan** [comment]
Yes, they are evenly distributed. We only sample along the X and Y axes, assuming the ground is a plane, so the Z-axis values are the same. Therefore, 1024 = 64 × 64. While there may be an initial error, the network can iteratively learn to optimize the poses and correct this discrepancy.

**Comment by gpastal24** [comment]
Ok thank you.
One last question, I have the same problem as in issue https://github.com/XunshanMan/MVGFormer/issues/5

I stored the model weights in models dir. strict is set to true but i the same results as in the aforementioned issue.
The dataset is preprocessed as in Faster Voxel Pose, which should be the same as in VoxelPose.

The command i run for evaluation is the following 

`
python run/validate_3d.py --cfg configs/panoptic/knn5-lr4-q1024.yaml --model_path models/mvgformer_q1024_model.pth.tar TEST.BATCH_SIZE=1
`

**Comment by XunshanMan** [comment]
Oh this is weired since I double checked before uploading. But I think I need to recheck now. 
We follow VoxelPose data preprocessing but I am not sure if Faster VoxelPose's data is the same. 
Did you download the same backbone too? We do not use the heat map, but use the last layer output of the 2D backbone to extract features. The corresponding code is here:
* https://github.com/XunshanMan/MVGFormer/blob/master/lib/models/dq_transformer.py#L341

I'll also check again by myself as soon I get some times. Thank you!

**Comment by gpastal24** [comment]
Hello, I tried yesterday to investigate further what is going on. I have uploaded some batch data in the following link.

[https://drive.google.com/drive/folders/1dnW_wjyMrFdzQeaGTpK6-H3ZaEyyLPIg?usp=sharing](url)

It appears to me that either the backbone is not producing correct features (i have used validate_3d as it is, but also with loading the 2D pretrained  backbone weights, which are not loaded by default, maybe it is an error?), either the model weights produce gibberish results, or there is something wrong with the camera params (I think this is unlikely). I have used both cuda 11.7 and 12.1, nothing changed. Could the error be traced to the deformable module, I have seen you have used cuda11.0. Lastly, when training, the model does not learn

**Comment by XunshanMan** [comment]
**Lastly, when training, the model does not learn**

Could you offer more details, e.g., outputs, loss curves etc. for this part too?

**Comment by gpastal24** [comment]
@XunshanMan Thank you very much. I will try the steps you have defined and will let you know how it went. 
I used  wandb offline and I cant seem to find the loss curves right now. However, total loss was in the millions and not decreasing. At the end of the first epoch, during evaluation I got Runtime error, cause the there were no predictions and torch.cat failed I believe?

The Deformable package appears to be installed, I also changed the build arguments for RTX3060 after trying everything, but still no luck. 

Anyway, I am closing this for now. I will try the scripts you have provided and  let you know. Thanks again!
